app/mailers/user_mailer.rb

```diff
@@ -9,12 +15,11 @@ def notify_subscriber(group, user, message)
     mail(to: @user.email, subject: 'Epic Logger Subscriber notification')
   end
 
-  def member_invitation(website_id, email, website_member_id, inviter_id)
-    @user = User.find(inviter_id)
-    @website = Website.find(website_id)
-    @token = WebsiteMember.find(website_member_id).invitation_token
-    @email = email
-    mail(to: @email, subject: 'Epic Logger Invite Users')
+  def member_invitation(invite)
+    @invite = invite
+    @inviter = User.find(@invite.invited_by_id)
```

Comment:
You also need to add a `has_many` clause in the user.rb model.
http://apidock.com/rails/ActiveRecord/Associations/ClassMethods/belongs_to
http://apidock.com/rails/v4.2.1/ActiveRecord/Associations/ClassMethods/has_many